spec/factories/blood_sugars.rb

```diff
@@ -6,9 +6,20 @@
     device_created_at { Time.current }
     device_updated_at { Time.current }
     recorded_at { device_created_at }
-    facility
-    user
-    patient
+
+    association :facility, strategy: :create
```

Comment:
@prabhanshuguptagit Why these changes?

Reply:
Just to keep things consistent across `blood_sugars` and `blood_pressures`.